docs/src/main/asciidoc/_elytron/Elytron_WildFly_Java_Security_Manager.adoc

```diff
@@ -0,0 +1,327 @@
+
+==== WildFly Security Manager Debug logging
+Debug is enabled via the following CLI
+[source,sh,options="nowrap"]
+----
+/subsystem=logging/logger=org.wildfly.security.access:add
+/subsystem=logging/logger=org.wildfly.security.access:write-attribute(name=level,value=DEBUG)
+----
+A sample output will look like this:
+[source]
+----
+DEBUG [org.wildfly.security.access] (Batch Thread - 1) Permission check failed (permission "("java.util.PropertyPermission" "java.io.tmpdir" "read")" in code source "(vfs:/content/batch-processing.war/WEB-INF/classes <no signer certificates>)" of "ModuleClassLoader for Module "deployment.batch-processing.war" from Service Module Loader")
+----
+
+==== WildFly Security Manager Trace logging
+You might have a case where you need to find out what exactly caused a certain permission request. Or you have an error case with an `null` classloader or `null` codesource. To get a stacktrace you can enable `org.wildfly.security.access` on log level `TRACE`.
+
+[source,sh,options="nowrap"]
+----
+/subsystem=logging/logger=org.wildfly.security.access:add
+/subsystem=logging/logger=org.wildfly.security.access:write-attribute(name=level,value=TRACE)
+----
+Attention: This generates a lot of log output and has a severe performance impact. It's not intended for permanent activation in development or testing but for special cases only.
+
+==== Logging Profiles
+If you are using one or more logging profiles it is strongly recommended to configure the `org.wildfly.security.access` logger for the server _AND_ all logging profiles. If not, you might miss a relevant output.
+
+[[Elytron_JSM_min_max_permissions]]
+=== Minimum and maximum permissions
+The ```security-manager``` subsystem configures a maximum-set with the ```AllPermission``` by default:
+[source,xml,options="nowrap"]
+----
+            <deployment-permissions>
+                <maximum-set>
+                    <permission class="java.security.AllPermission"/>
+                </maximum-set>
+            </deployment-permissions>
+----
+If you cannot configure a third-party blackbox deployment unit or you want to share a common set of permissions across multiple deployments you can add a ```minimum-set``` of permissions.
+You could modify the ```maximum-set```, remove the AllPermission and setup further restrictions to permissions you are willing to grant to deployments.
+
+[[Elytron_JSM_permissions_xml_property_replacement]]
+=== Property replacement in permissions.xml
+https://github.com/wildfly/wildfly-proposals/blob/main/ee/WFCORE_2147_Wildfly_core_descriptor_based_property_replacement.adoc#enabledisable-descriptor-based-property-replacement-for-files-parsed-by-wildfly-core[Expression resolution in EE security manager deployment descriptors] (permissions.xml and jboss-permissions.xml) is supported since WildFly 19. You can use the familiar ```${foo:true}``` syntax in these deployment descriptors to allow customization of settings at runtime.
```

Comment:
We should use link to the documentation and not link to the proposal if available

Reply:
Thanks, found it - fixed.